doc/api/child_process.md

```diff
@@ -131,6 +131,9 @@ exec('"my script.cmd" a b', (err, stdout, stderr) => {
 <!-- YAML
 added: v0.1.90
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/PR-XXXXX
```

Comment:
These have to be changed before / while landing to `https://github.com/nodejs/node/pull/21316`.